test/e2e/specs/editor/various/editor-modes.spec.js

```diff
@@ -155,4 +155,35 @@ test.describe( 'Editing modes (visual/HTML)', () => {
 			},
 		] );
 	} );
+
+	test( 'should reparse changes from code editor', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		// Open the code editor.
+		await pageUtils.pressKeys( 'secondary+M' );
+
+		const textbox = page.getByRole( 'textbox', {
+			name: 'Type text or HTML',
+		} );
+
+		// Change content by typing.
+		await textbox.fill( '' );
```

Comment:
Why is this necessary?

Reply:
There's a beforeEach that fills in content